dataactbroker/scripts/dedup_duns_export.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import boto
+import os
+import pandas as pd
+import csv
+from datetime import datetime
+
+from dataactvalidator.health_check import create_app
+from dataactcore.logging import configure_logging
+from dataactcore.config import CONFIG_BROKER
+
+logger = logging.getLogger(__name__)
+
+# CSV column header name in DUNS file
+column_headers = [
+    "DUNS",
+    "Registration_Date",
+    "Expiration_Date",
+    "Last_Update_Date",
+    "Activation_Date",
+    "Legal_Business_Name"
+]
+
+
+def generate_dedup_export(duns_file, dedup_export_path):
+    """Generate dedup duns export"""
```

Comment:
Please refer to `StyleGuide.md` for comment standards (this applies to all comments/files in the PR)

Reply:
Updated.